### ISSUE TITLE
Big Sur not working!

### ISSUE THREAD

**Issue by warshamanage** [objective-see/LuLu#264]
Even allow the google chrome and firefox, cannot browse the internet when LULU enabled.

**Comment by lionep** [comment]
Can confirm here : 
Even when starting disabled, I need to enable then disable it again to get internet access on browser (maybe this is a second issue).

**Comment by parth2187** [comment]
I can confirm this issue. I thought my wifi was not working but then I realized that my phone is currently connected to the same wifi network and getting notifications. I had to quit LuLu forcefully to restore the internet connection. 

I wish I could provide some logs but I am afraid that I don't know how to report bugs and logs.

**Comment by Carrotcruncher** [comment]
I had this issue on Big Sur with Apple silicon, my solution was to reboot into recovery by holding down touch i.d and then change security level to insecure which allows changes to kernel by third party apps

**Comment by Carrotcruncher** [comment]
After a recent restart today Lulu is now blocking internet despite all rules being correct, I cannot browse nor can any program access the internet, this is with apple silicon

**Comment by warshamanage** [comment]
Hello all, please try this method.
First of all, allow all the rules for internet access. then uninstall the LULU using menubar>icon>rightclick>uninstall>restart>download fresh copy>install again>thanks me later..

**Comment by Carrotcruncher** [comment]
> Hello all, please try this method.
> First of all, allow all the rules for internet access. then uninstall the LULU using menubar>icon>rightclick>uninstall>restart>download fresh copy>install again>thanks me later..

Doesn't work, same issue

**Comment by pastrychef** [comment]
> After a recent restart today Lulu is now blocking internet despite all rules being correct, I cannot browse nor can any program access the internet, this is with apple silicon


Same issue here.  Lulu 2.0.0 blocks all network activity for me on Big Sur.

**Comment by jchhenderson** [comment]
Yes, same for me on Big Sur.

2.0.0 blocks everything
1.2.3 is somehow broken (Existing rules don't show.  Menu items are missing.  Had to manually uninstall).

Sadly, both have had to be removed.

**Comment by jchhenderson** [comment]
> Hello all, please try this method.
> First of all, allow all the rules for internet access. then uninstall the LULU using menubar>icon>rightclick>uninstall>restart>download fresh copy>install again>thanks me later..

This did not work for me.

**Comment by fate8383** [comment]
The same issue here, both macs (mojave and big sur) upgraded from v123 to v2. Cannot connect to internet without disabling LULU. Hopefully this can be fixed otherwise no point using LULU as it does not work.

**Comment by dchakro** [comment]
> Key thing is getting lulu app in "running" state under settings>network, it was on "starting" state before and it is not good.

@fate8383  Absolutely, this is what I narrowed down it to as well. But I could you walk us through on how you fixed it for yourself? i.e. when you clicked "allow all blocked rules", did the status in Settings -> Network, change to "running" for you? Also you didn't mention that after re-install did you block you're desired apps (at all)? Did you block them manually again or did you restore the rules?

**Comment by fate8383** [comment]
ok i didn't use allow all blocked rules, so I have had few rules of apps blocked, so individually i changed all to allow or simply i removed the blocked rule(s). Then uninstalled, rebooted, reinstalled, and it started working. Of course afterward i blocked again those rules I removed or changed to allowed. Before you are doing anything disable or exit your VPNs, after LULU app is working (running state) then you can also use your VPN apps without a problem together with LULU. Hope this helps

**Comment by TheMen4ce** [comment]
For me it only works on the first install. When I then reboot my machine, all outgoing connections are blocked until I quit Lulu.

Running MacOS 11.1 (20C69)
Lulu 2.0.0

**Comment by sahilc0** [comment]
Thank you @fate8383! That process worked for me!
I enabled my VPN after reinstalling and it works now.

**Comment by jchhenderson** [comment]
I tried again, fresh install etc.  Everything is still blocked.  I don't understand it and have had to uninstall to write this.

**Comment by gyrex** [comment]
I'm having the same issue with Big Sur and Lulu v2.0.0. If Lulu is enabled, internet access is completely blocked. The only way to access the internet is to disable LuLu. Does anyone know if there's a fix to this?

**Comment by objective-see** [comment]
Sorry about this 😔

Can you try the latest version of LuLu (v2.3.1): https://github.com/objective-see/LuLu/releases/tag/v2.3.1

...and also make sure you are on at least on version macOS 11.2, as Apple fixed lots of bugs in their Network Extension Framework (that LuLu leverages). 

If it's still broken, please let me know! 🙏🏽

**Comment by jchhenderson** [comment]
no worries and installing it now!

LuLu worked fine except during the 11.2.2 critical update process, so not sure how I am going to be able to prove this new version one way or the other though.

**Comment by 10SexyApples** [comment]
I'm running Catalina and LuLu (v2.3.1) and experiencing the same issues with having to disable in order to access internet. Oddly enough though, I installed LuLu initially on a clean install about half way through setting the machine up and it worked fine. However, I made the decision to uninstall and continue on with my installation saving LuLu for last so that all of my core safe apps would be auto included in the rules. I didn't begin to experience this issue until the reinstallation.

**Comment by doubleopinter** [comment]
I just wanted to throw my two cents in and note that I am having the same issue, just removed Lulu unfortunately. I'm on an M1 MBP, latest all the things. When lulu starts I can't connect to anything.
